changelogs/fragments/6919.yml

```diff
@@ -0,0 +1,2 @@
+feat:
+- Modify the adding sample data part for timeline ([#6919](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919))
```

Comment:
The entry should describe the outcome not what was done.

Reply:
Thanks Miki. How about "Enable the addition of sample data for timeline visualizations with MDS enabled. "